k-distribution/tests/regression-new/io-ocaml/1.test.out

```diff
@@ -0,0 +1,3 @@
+<k>
```

Comment:
should not have its own set of output files. You should just set TESTDIR in the makefile to point to the other directory.